src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+
+      <p className="Person__partner">
+        {!isMarried
+          ? 'I am not married'
+          : `${partnerName} is my ${partner}`}
```

Comment:
No need to use reverse condition. This way it's more simple.
```suggestion
        {isMarried
          ? `${partnerName} is my ${partner}`
          : 'I am not married'}
```